static/src/inline-svgs/journalism/audio-player/pause-btn.svg

```diff
@@ -1 +1 @@
-<svg width="78" height="78" viewBox="0 0 78 78"><title>pause btn</title><g transform="translate(1 1)" fill="none" fill-rule="evenodd"><circle stroke="#00B2FF" stroke-width="2" fill="#00B2FF" cx="38" cy="38" r="38"/><path d="M23.886 18.457h8.686v39.086h-8.686zm18.457 0h8.686v39.086h-8.686z" fill="#FFF"/></g></svg>
\ No newline at end of file
+<svg width="78" height="78" viewBox="0 0 78 78"><title></title><g transform="translate(1 1)" fill="none" fill-rule="evenodd"><circle stroke="#00B2FF" stroke-width="2" fill="#00B2FF" cx="38" cy="38" r="38"/><path d="M23.886 18.457h8.686v39.086h-8.686zm18.457 0h8.686v39.086h-8.686z" fill="#FFF"/></g></svg>
```

Comment:
You might as well drop all the `title` elements then, I'm not sure what the behaviour is with empty elements across browsers.